frontend/package.json

```diff
@@ -38,9 +38,9 @@
     "storybook": "start-storybook -p 6006 -s ./config"
   },
   "dependencies": {
-    "@openshift/dynamic-plugin-sdk": "^1.0.0-alpha12",
-    "@openshift/dynamic-plugin-sdk-utils": "^1.0.0-alpha17",
     "@patternfly/patternfly": "4.224.2",
+    "@openshift/dynamic-plugin-sdk": "^3.0.0",
+    "@openshift/dynamic-plugin-sdk-utils": "^1.2.0",
```

Comment:
Why did you move these -- this should be in alphabetical order? Did you manually do this because I would assume NPM sorts them...

Reply:
Yes, my bad, sorting it again.